D.S Linkedlist/Linkedlist-hw1.php

```diff
@@ -0,0 +1,105 @@
+<?

```

Comment:
@Misakiiii-333 Please change the open tag of php in the right way, and make sure you make a test before susbmit. Thank you!